vpr/src/base/clustered_netlist.cpp

```diff
@@ -183,18 +189,25 @@ ClusterNetId ClusteredNetlist::create_net(const std::string name) {
 
     if (net_id == ClusterNetId::INVALID()) {
         net_id = Netlist::create_net(name);
-        net_is_global_.push_back(false);
+        net_is_ignored_.push_back(false);
+        is_global_net_.push_back(false);
     }
 
     VTR_ASSERT(validate_net_sizes());
 
     return net_id;
 }
 
-void ClusteredNetlist::set_net_is_global(ClusterNetId net_id, bool state) {
+void ClusteredNetlist::set_net_is_ignored(ClusterNetId net_id, bool state) {
+    VTR_ASSERT(valid_net_id(net_id));
+
+    net_is_ignored_[net_id] = state;
+}
+
+void ClusteredNetlist::set_is_global_net(ClusterNetId net_id, bool state) {
```

Comment:
Before merge, convert set_is_global_net/is_global_net back to set_net_is_global/net_is_global.

That ensures the netlist function naming convention is consistent.